docs/macos_install_guide.rst

```diff
@@ -67,10 +67,6 @@ If you want to use the Web-UI you additionally need node.js::
 
     $ brew install node
```

Comment:
Do we even need node anymore @kelsos?

Reply:
Since we used it only to compile the WebUI, if I am not mistaken, then I would say no. The webui python package already contains the final compiled resources.

Reply:
we shouldn't